packages/strapi-plugin-content-type-builder/admin/src/components/LiTableList/index.js

```diff
@@ -0,0 +1,55 @@
+/**
+*
+* LiTableList
+*
+*/
+
+import React from 'react';
+import { includes, startCase } from 'lodash';
+import styles from 'components/TableList/styles.scss';
+import { router } from 'app';
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+
+class LiTableList extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  edit = () => {
+    console.log('edit', this.props.rowItem.name);
+  }
+
+  delete = () => {
+    console.log('delete', this.props.rowItem.name);
+  }
+
+  goTo = (e) => {
```

Comment:
You should use `e.preventDefault()` and `e.stopPropagation()` instead of using a condition to ensure the click has been done on the right DOM element.